mcmod-info/src/main/java/org/spongepowered/plugin/meta/McModInfo.java

```diff
@@ -54,6 +56,7 @@
  * Represents a serializer for {@link PluginMetadata} for the
  * {@code mcmod.info} file format.
  */
+@DefaultQualifier(NonNull.class)
```

Comment:
Isn't this redundant given the one in `package-info.java` (although located in a different module)?